packages/loaders/src/Dots.example.md

```diff
@@ -0,0 +1,179 @@
+The `<Dots />` component includes several accessibility
+and usability features:
+
+- Inherits the `font-size` and `color` properties
+  from it's parent element and additionally allows local overrides
+- Delays the initial render of the component to reduce UI shifting
+  in normal loading conditions
+- Includes accessibility attributes to identify as an `indeterminate progress bar`
+
+### Basic Usage
+
+```jsx
+const { zdColorBlue500 } = require('@zendeskgarden/css-variables');
+
+<Dots color={zdColorBlue500} size="50px" />;
+```
+
+### Loading Button
+
+The `Dots` loader inherits it's size and color from it's parent element by default.
+
+For usages where a rendering delay is not necessary, pass `[delayMS={0}]`
+
+```jsx
+const { Button } = require('@zendeskgarden/react-buttons/src');
+
+class ButtonExample extends React.Component {
+  constructor(...args) {
+    super(...args);
+
+    this.state = {
+      isLoading: false
+    };
+  }
+
+  componentDidMount() {
+    this.interval = setInterval(() => {
+      this.setState(prevState => ({
+        isLoading: !prevState.isLoading
+      }));
+    }, 2000);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.interval);
+  }
+
+  render() {
+    return (
+      <Button primary>
+        {!this.state.isLoading && 'This button has a delayed loading state'}
+        {this.state.isLoading && <Dots delayMS={0} />}
+      </Button>
+    );
+  }
+}
+<ButtonExample />;
+```
+
+### Advanced Usage
+
+```jsx
+const { RangeField, Label, Range } = require('@zendeskgarden/react-ranges/src');
+const {
+  SelectField,
+  Label: SelectLabel,
+  Select,
+  Item
+} = require('@zendeskgarden/react-select/src');
+
+const colors = {
+  'BLUE-500': '#337FBD',
+  'GREY-500': '#87929D',
```

Comment:
Is there a way we can we depend on `css-variables` for these values?

Reply:
Now added